plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchange.java

```diff
@@ -43,10 +43,10 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
-import java.util.UUID;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ThreadLocalRandom;
```

Comment:
from cmt msg

> such that the first two chars has as many combinations as possible.

why?

Reply:
Ah already merged. This is mainly for S3. Users can talk to AWS to set up manual partitioning for S3 buckets, and they do that base on the first n chars. In our case, it's the first 2.